CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.62.0
+
+- feat(integrations): Add `ContextLines` integration for html-embedded JS stack frames (#8699)
```

Comment:
m: I think we can have a call out for this - maybe some folks will find it interesting.